Makefile

```diff
@@ -14,19 +14,23 @@ all: release
 
 $(VENV):
 	pip3 install virtualenv
-	virtualenv $(VENV) --python=python3.8
-	$(PIP) install .[dev]
+	virtualenv $(VENV) --python=python3.9
 	$(PIP) install setuptools -U
 
-
 .PHONY: clean
 clean:
 	rm -rf $(VENV) *.egg-info .eggs .egg htmlcov build dist packages .build .tmp .tox __pycache__
 
 .PHONY: deps
-deps: $(VENV)
+deps: $(VENV) install-packages
+	@echo "Installing all dependencies..."
 	$(PIP) install .[dev]
 
+.PHONY: install-packages
+install-packages:
+	@echo "Installing kql and kibana packages..."
+	$(PIP) install -e tools/kql
+	$(PIP) install -e tools/kibana
```

Comment:
Now that we've decoupled from the base pyproject.toml, we still want to make it easy to install our requirements.